packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.types.ts

```diff
@@ -176,6 +177,11 @@ export interface IDetailsRowBaseProps
    */
   className?: string;
 
+  /**
+   * Props that should be passed into the focus zone, will not override any default props
+   */
+  focusZoneProps?: IFocusZoneProps;
```

Comment:
Is this consistent with our other components? I feel like other components do let users override defaults and that's one primary reason for exposing them.